### ISSUE TITLE
Main menu a11y issues

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9254]
# Issue

Container element is empty

Some roles are designed to contain other roles. This element has this type of role, but it doesn't contain any [required-owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain).

Seems to be related to the main menu on SiteNow sites. Not seeing a `role="list"` though.

https://my2.siteimprove.com/Inspector/998093/A11y/Page?pageId=24733541800&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/cXpidjV2ZEc=;OTNiODc2ZDg=
